src/components/inboxComponent/chatroom.tsx

```diff
@@ -20,30 +15,27 @@ const SelectedChatroom: React.FC = () => {
   const { id } = useParams<RouteParams>();
   const [message, setMessage] = useState('');
   const { user, inbox } = useSelector((state: StoreState) => state);
-  const { chatroom } = inbox;
-  const dispatch = useDispatch();
+  const chatroom = inbox.chatrooms.find(room => room.id === id);
```

Comment:
we should probably have some logic inside of a useEffect to grab the inbox... what if they refresh on this page?

Reply:
tested out the refresh functionality, it doesnt load the page until the rest of the app loads